specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -5530,6 +5530,207 @@
       },
       "description": "Ranking function based on the Okapi BM25 similarity algorithm. BM25 is a TF-IDF-like algorithm that includes length normalization (controlled by the 'b' parameter) as well as term frequency saturation (controlled by the 'k1' parameter)."
     },
+    "VectorSearch": {
+      "type": "object",
+      "properties": {
+        "profiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchProfile"
+          },
+          "description": "Defines combinations of configurations to use with vector search."
+        },
+        "algorithms": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchAlgorithmConfiguration"
+          },
+          "description": "Contains configuration options specific to the algorithm used during indexing and/or querying."
+        }
+      },
+      "description": "Contains configuration options related to vector search."
+    },
+    "VectorSearchProfile": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "externalDocs": {
+            "url": "https://docs.microsoft.com/rest/api/searchservice/Naming-rules"
+          },
+          "type": "string",
+          "description": "The name to associate with this particular vector search profile.",
+          "x-nullable": false
+        },
+        "algorithm": {
+          "type": "string",
+          "description": "The name of the vector search algorithm configuration that specifies the algorithm and optional parameters.",
+          "x-nullable": false
+        }
+      },
+      "required": [
+        "name",
+        "algorithm"
+      ],
+      "description": "Defines a combination of configurations to use with vector search."
+    },
+    "VectorSearchAlgorithmConfiguration": {
+      "type": "object",
+      "discriminator": "kind",
+      "properties": {
+        "name": {
+          "externalDocs": {
+            "url": "https://docs.microsoft.com/rest/api/searchservice/Naming-rules"
```

Comment:
```suggestion
            "url": "https://learn.microsoft.com/rest/api/searchservice/Naming-rules"
```

Reply:
and same for all other places

Reply:
@DominikMe I have updated the url's to the new path.